src/components/ecommerce/validators.ts

```diff
@@ -0,0 +1,15 @@
+export const CardNumberValidator = (value: string): boolean => {
+  return value.length === 19;
+};
+
+export const ExpirationDateValidator = (value: string): boolean => {
+  return value.length === 5;
+};
+
+export const CvvValidator = (value: string): boolean => {
+  return value.length === 3;
+};
+
+export const CardholderNameValidator = (value: string): boolean => {
+  return /^$|^[a-zA-ZčČćĆđĐšŠžŽ-]+ [a-zA-ZčČćĆđĐšŠžŽ-]+$/.test(value) && value.length !== 0;
```

Comment:
prefer refactoring to use existing regex for name

Reply:
I don't know how to integrate it together.